src/v3/test/integration/util/setup.tsx

```diff
@@ -23,6 +23,7 @@ import { Widget } from '../../../src/components/Widget';
 type Options = CreateAuthClientOptions & {
   widgetOptions?: Partial<WidgetOptions>;
 };
+jest.unmock('okta');
```

Comment:
can you leave a comment about why we unmock the 'okta' package here?

Reply:
restoreMock cannot work?

Reply:
Absolutely! It is because in the integration tests, it makes sense (to me at least) to display the translated string instead of an i18n key. Let me know if I am wrong though.

Reply:
@shuowu-okta `restoreAllMocks` does not work unfortunately, I added a comment above this line for future reference including a link to jest docs. 
https://jestjs.io/docs/manual-mocks#examples

Reply:
Sorry @glenfannin-okta I should have been clearer -- I understand why we should unmock here but can you put a comment in the code saying we want to see the translated strings in the integration tests?

Reply:
Ohh, sorry about that, done.

Reply:
oh right, if it can work you won't need add any code, it's already in jest.config